easybuild/easyconfigs/u/umesimd/umesimd-0.8.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'CMakeMake'
+name = 'umesimd'
+version = '0.8.1'
+
+description = """
+UME::SIMD is an explicit vectorization library. The library defines homogeneous interface for accessing functionality of SIMD registers of AVX, AVX2, AVX512 and IMCI (KNCNI, k1om) instruction set.
+"""
+
+homepage = 'https://github.com/edanor/umesimd'
+source_urls = ['https://github.com/edanor/umesimd/archive']
+sources = ['v%(version)s.zip']
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+#test_cmd = 'cd %(builddir)s/%(name)s-%(version)s/unittest && '
+#runtest = 'CXX=g++ ISA=avx2 BUILD=release_O3 make executable -k && ./gcc.out'
+
```

Comment:
If we can't run these it should be cleared out instead, but can't it do
https://github.com/edanor/umesimd/blob/master/.travis.yml#LL309C1-L311C50
?